component/loki/source/kubernetes/kubetail/tailer.go

```diff
@@ -79,7 +79,7 @@ func newLabelSet(l labels.Labels) model.LabelSet {
 }
 
 var retailBackoff = backoff.Config{
-	MinBackoff: time.Second,
+	MinBackoff: 0,
```

Comment:
I set this back to zero so the very first retry is instantaneous, which is useful to make sure that the maximum lifetime of the tailer doesn't have to wait a second before reestablishing the connection.